src/Store/StoreThread.re

```diff
@@ -191,16 +191,17 @@ let start =
 
   let subscriptions = (state: Model.State.t) => {
     let syntaxSubscription =
-      Feature_Syntax.subscription(
-        ~configuration=state.configuration,
-        ~enabled=shouldSyntaxHighlight,
-        ~quitting=state.isQuitting,
-        ~languageInfo,
-        ~setup,
-        ~tokenTheme=state.tokenTheme,
-        state.syntaxHighlights,
-      )
-      |> Isolinear.Sub.map(msg => Model.Actions.Syntax(msg));
+      shouldSyntaxHighlight && !state.isQuitting
+        ? Feature_Syntax.subscription(
+            ~configuration=state.configuration,
```

Comment:
The only `configuration` we use is whether experimental tree-sitter is enabled, so we can just pass that directly